.gitignore

```diff
@@ -28,6 +28,8 @@ Testing/
 target/
 cluster.id
 pids/
+bin/
```

Comment:
We strongly discourage to build the project directly in the source tree.

Reply:
Sounds good. we should make nebula under `build` directory. closed